src/plugins/textReplace/index.tsx

```diff
@@ -333,8 +415,30 @@ const TEXT_REPLACE_RULES_CHANNEL_ID = "1102784112584040479";
 
 export default definePlugin({
     name: "TextReplace",
-    description: "Replace text in your messages. You can find pre-made rules in the #textreplace-rules channel in Vencord's Server",
-    authors: [Devs.AutumnVN, Devs.TheKodeToad],
+    description: "Replace text in your messages.",
+    authors: [Devs.TheKodeToad, Devs.skykittenpuppy, Devs.AutumnVN],
```

Comment:
```suggestion
    authors: [Devs.AutumnVN, Devs.TheKodeToad, Devs.skykittenpuppy],
```
dont change authors order